Default name for Web Application conflicts with ASP.NET 5 WebApplication class

In RC1 ASP.NET 5 has a class called [WebApplication](https://github.com/aspnet/Hosting/blob/dev/src/Microsoft.AspNet.Hosting/WebApplication.cs). 

The default name of a Web Application project created with `yo aspnet` is also WebApplication.

Because of the conflict, we get a startup exception:
```
Application startup exception: Microsoft.Dnx.Compilation.CSharp.RoslynCompilationException: /Users/mg/Src/asp/WebApplication/Startup.cs(102,51): DNX,Version=v4.5.1 error CS0234: The type or namespace name 'Run' does not exist in the namespace 'WebApplication' (are you missing an assembly reference?)
  at Microsoft.Dnx.Compilation.CSharp.RoslynProjectReference.Load (System.Reflection.AssemblyName assemblyName, IAssemblyLoadContext loadContext) [0x00000] in <filename unknown>:0
  at Microsoft.Dnx.Compilation.CompilationEngine.LoadProject (Microsoft.Dnx.Runtime.Project project, System.Runtime.Versioning.FrameworkName targetFramework, System.String aspect, IAssemblyLoadContext loadContext, System.Reflection.AssemblyName assemblyName) [0x00000] in <filename unknown>:0
  at Microsoft.Dnx.Runtime.Loader.ProjectAssemblyLoader.Load (System.Reflection.AssemblyName assemblyName, IAssemblyLoadContext loadContext) [0x00000] in <filename unknown>:0
  at Microsoft.Dnx.Runtime.Loader.ProjectAssemblyLoader.Load (System.Reflection.AssemblyName assemblyName) [0x00000] in <filename unknown>:0
  at Microsoft.Dnx.Host.LoaderContainer.Load (System.Reflection.AssemblyName assemblyName) [0x00000] in <filename unknown>:0
  at Microsoft.Dnx.Host.DefaultLoadContext.LoadAssembly (System.Reflection.AssemblyName assemblyName) [0x00000] in <filename unknown>:0
  at Microsoft.Dnx.Runtime.Loader.AssemblyLoaderCache.GetOrAdd (System.Reflection.AssemblyName name, System.Func`2 factory) [0x00000] in <filename unknown>:0
  at Microsoft.Dnx.Runtime.Loader.LoadContext.LoadAssemblyImpl (System.Reflection.AssemblyName assemblyName) [0x00000] in <filename unknown>:0
  at Microsoft.Dnx.Runtime.Loader.LoadContext.ResolveAssembly (System.Object sender, System.ResolveEventArgs args) [0x00000] in <filename unknown>:0
  at System.AppDomain.DoAssemblyResolve (System.String name, System.Reflection.Assembly requestingAssembly, Boolean refonly) [0x00000] in <filename unknown>:0
  at (wrapper managed-to-native) System.AppDomain:LoadAssembly (System.AppDomain,string,System.Security.Policy.Evidence,bool)
  at System.AppDomain.Load (System.Reflection.AssemblyName assemblyRef, System.Security.Policy.Evidence assemblySecurity) [0x00000] in <filename unknown>:0
  at (wrapper remoting-invoke-with-check) System.AppDomain:Load (System.Reflection.AssemblyName,System.Security.Policy.Evidence)
  at System.AppDomain.Load (System.Reflection.AssemblyName assemblyRef) [0x00000] in <filename unknown>:0
  at (wrapper remoting-invoke-with-check) System.AppDomain:Load (System.Reflection.AssemblyName)
  at System.Reflection.Assembly.Load (System.Reflection.AssemblyName assemblyRef) [0x00000] in <filename unknown>:0
  at Microsoft.AspNet.Hosting.Startup.StartupLoader.FindStartupType (System.String startupAssemblyName, IList`1 diagnosticMessages) [0x00000] in <filename unknown>:0
  at Microsoft.AspNet.Hosting.Internal.HostingEngine.EnsureStartup () [0x00000] in <filename unknown>:0
  at Microsoft.AspNet.Hosting.Internal.HostingEngine.EnsureApplicationServices () [0x00000] in <filename unknown>:0
  at Microsoft.AspNet.Hosting.Internal.HostingEngine.BuildApplication () [0x00000] in <filename unknown>:0
```

--- a/templates/projects/empty/Startup.cs
+++ b/templates/projects/empty/Startup.cs
@@ -29,6 +29,6 @@ namespace <%= namespace %>
         }
 
         // Entry point for the application.
-        public static void Main(string[] args) => WebApplication.Run<Startup>(args);
+        public static void Main(string[] args) => Microsoft.AspNet.Hosting.WebApplication.Run<Startup>(args);
     }
 }

--- a/templates/projects/web/Startup.cs
+++ b/templates/projects/web/Startup.cs
@@ -99,6 +99,6 @@ namespace <%= namespace %>
         }
 
         // Entry point for the application.
-        public static void Main(string[] args) => WebApplication.Run<Startup>(args);
+        public static void Main(string[] args) => Microsoft.AspNet.Hosting.WebApplication.Run<Startup>(args);
     }
 }

--- a/templates/projects/webapi/Startup.cs
+++ b/templates/projects/webapi/Startup.cs
@@ -44,6 +44,6 @@ namespace <%= namespace %>
         }
 
         // Entry point for the application.
-        public static void Main(string[] args) => WebApplication.Run<Startup>(args);
+        public static void Main(string[] args) => Microsoft.AspNet.Hosting.WebApplication.Run<Startup>(args);
     }
 }

--- a/templates/projects/webbasic/Startup.cs
+++ b/templates/projects/webbasic/Startup.cs
@@ -60,6 +60,6 @@ namespace <%= namespace %>
         }
 
         // Entry point for the application.
-        public static void Main(string[] args) => WebApplication.Run<Startup>(args);
+        public static void Main(string[] args) => Microsoft.AspNet.Hosting.WebApplication.Run<Startup>(args);
     }
 }

